translator/lib/SPIRV/SPIRVReader.cpp

```diff
@@ -56,6 +56,7 @@
 #include "llvm/ADT/StringSwitch.h"
 #include "llvm/BinaryFormat/Dwarf.h"
 #include "llvm/IR/Constants.h"
+#include "llvm/IR/CFG.h"
```

Comment:
why do we need this header file? which function require this? thanks!

Reply:
It's for `successors` and `predecessors`